src/main/java/by/exadel/internship/controller/LocationController.java

```diff
@@ -0,0 +1,31 @@
+package by.exadel.internship.controller;
+
+import by.exadel.internship.dto.locationDTO.CityDTO;
+import by.exadel.internship.dto.locationDTO.CountryDTO;
+import by.exadel.internship.service.LocationService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/location")
```

Comment:
@OlgaEvdokimova Question: should these endpoints be a public with unauthorized access?

Reply:
Yes, they will be called from Registration public form

Reply:
Great, let's wait #35 merging and it need to add your urls to white lists for public urls